css/src/components/buttons.scss

```diff
@@ -76,6 +75,14 @@
 				}
 			}
 
+			@include focus-visible {
```

Comment:
Comment: not related to the changes here, but it seems like the filled button on this page is missing a visible focus ring https://design.docs.microsoft.com/pulls/371/components/button.html

Reply:
Ah good catch, you and @BenDMyers have found the same thing

Reply:
Addressing in a new one

Reply:
https://ceapex.visualstudio.com/Engineering/_workitems/edit/565910